src/components/Book/BookAbout/BookAbout.module.scss

```diff
@@ -0,0 +1,58 @@
+@import 'variables.scss';
+
+.c-book-details {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+
+  width: 100%;
+
+  margin-top: $space-xl;
+  margin-bottom: $space-xl;
+
+  &__image-holder {
+    display: flex;
+    justify-content: center;
+
+    width: 30%;
+    height: 50vh;
+
+    object-fit: contain;
+  }
+
+  &__image {
+    width: 100%;
+  }
+
+  &__about {
+    width: 30%;
+  }
+
+  &__author {
+    margin-top: $space-s;
+
+    font-size: $fs-large;
+
+    color: $color-grey;
```

Comment:
Here `font-size` and `color` should be grouped together as they are changing font